packages/terra-embedded-content-consumer/tests/wdio/embedded-content-consumer-spec.js

```diff
@@ -1,51 +1,44 @@
 Terra.describeViewports('Embedded Content Consumer', ['tiny', 'large'], () => {
-  describe('default', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/basic-consumer');
-    });
-
-    Terra.it.matchesScreenshot();
+  it('displays default embedded content consumer', () => {
+    browser.url('/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/basic-consumer');
+    Terra.validates.screenshot('default');
   });
 
   describe('custom-event', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/custom-event-consumer');
+    it('displays with custom events', () => {
+      browser.url('/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/custom-event-consumer');
       browser.waitForExist('#CustomEvent');
+      Terra.validates.screenshot('custom event');
     });
 
-    Terra.it.matchesScreenshot();
-
     it('Provider triggers EventA message', () => {
       const myFrame = browser.element('iframe[src="/#/raw/provider/terra-embedded-content-consumer/embedded-content-consumer/providers/custom-event-provider"]').value;
       browser.frame(myFrame);
 
       browser.click('#EventA');
       browser.frameParent();
       browser.moveToObject('#root', 0, 0);
+      Terra.validates.screenshot('EventA');
     });
-
-    Terra.it.matchesScreenshot('EventA');
   });
 
   describe('custom-events', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/custom-events-consumer');
+    it('displays with custom event', () => {
+      browser.url('/raw/tests/terra-embedded-content-consumer/embedded-content-consumer/consumers/custom-events-consumer');
       browser.waitForExist('#CustomEvents');
+      Terra.validates.element('custom event');
     });
 
-    Terra.it.matchesScreenshot();
-
     it('Provider triggers EventA message', () => {
       const myFrame = browser.element('iframe[src="/#/raw/provider/terra-embedded-content-consumer/embedded-content-consumer/providers/custom-events-provider"]').value;
```

Comment:
```suggestion
      const myFrame = browser.element('iframe[src="/raw/provider/terra-embedded-content-consumer/embedded-content-consumer/providers/custom-events-provider"]').value;
```

Reply:
Screenshots were failing when I  had made this change

Reply:
> Screenshots were failing when I had made this change

Why were they failing?

Is it because the test routes are still using the hash?

https://github.com/cerner/terra-framework/blob/64adc177e1c4729ffa23c5b6e77b2f75b1c1ba3a/packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/consumers/BasicConsumer.test.jsx#L14

Reply:
Yes, fixed here 7b428c6